lightly/cli/serve_cli.py

```diff
@@ -0,0 +1,54 @@
+import sys
+from pathlib import Path
+
+import hydra
+
+from lightly.api import serve
+from lightly.cli._helpers import fix_hydra_arguments
+
+
+@hydra.main(**fix_hydra_arguments(config_path="config", config_name="lightly-serve"))
+def lightly_serve(cfg):
+    """Use lightly-serve to serve your data for interactive exploration.
+
+    Command-Line Args:
+        input_dir:
+            Path to the input directory.
+        lightly_dir:
+            Path to the Lightly directory.
+        host:
+            Hostname for serving the data (defaults to localhost).
+        port:
+            Port for serving the data.
+
+    Examples:
+        >>> lightly-serve input_dir=data/ lightly_dir=lightly/ port=8080
```

Comment:
best to use 4567 as example as afaik thats also the default in `set_local_datasource`

Reply:
3456

Reply:
There are no dashes? I.e. `--port=8000` etc?

Reply:
Not if we want to keep working with [hydra](https://hydra.cc/docs/advanced/override_grammar/basic/). The advantage of hydra is that you can use your own config files so you don't always have to remember which directories to mount.